_includes/releases/v22.1/v22.1.0-beta.1.md

```diff
@@ -6,90 +6,75 @@ Release Date: April 4, 2022
+- Do not block schema changes when executing core-style changefeeds. [#78360][#78360]
 
 <h3 id="v22-1-0-beta-1-sql-language-changes">SQL language changes</h3>
 
-- Add support for `ALTER DATABASE ... ALTER SUPER REGION`.  This command allows the user to change the regions of an existing super region.  
-	
-	For example, after successful execution of the following, super region `test1` will consist of three regions, `ca-central-1`, `us-west-1`, and `us-east-1`.
+- Added support for `ALTER DATABASE ... ALTER SUPER REGION`.  This command allows the user to change the regions of an existing super region. For example, after successful execution of the following, super region `test1` will consist of three regions, `ca-central-1`, `us-west-1`, and `us-east-1`.
 
-	`ALTER DATABASE db3 ALTER SUPER REGION "test1" VALUES "ca-central-1", "us-west-1", "us-east-1";`
+{% include_cached copy-clipboard.html %}
+```shell
+ALTER DATABASE db3 ALTER SUPER REGION "test1" VALUES "ca-central-1", "us-west-1", "us-east-1";
+```
 	
-	`Alter super region` follows the same rules as `add` or `drop` super region. [#78462][#78462]
-- The session variables `datestyle_enabled` and `intervalstyle_enabled`, and the cluster settings `sql.defaults.datestyle.enabled` and `sql.defaults.intervalstyle.enabled` no longer have any effect. After upgrading to v22.1, these settings are effectively always interpreted as "true." [#78455][#78455]
-- Stats compaction scheduled job no longer cause intent buildup. Release Justification: important stability fix to ensure jobs and scheduled jobs do not lock up when running stats compaction job. [#78565][#78565]
-- Bucket-count of hash-sharded index is now shown from the `crdb_internal.table_indexes` table. [#78625][#78625]
-- Implemented the `COPY FROM ... ESCAPE ...` syntax. [#78417][#78417]
-- Disabled index recommendations in `EXPLAIN` output for `REGIONAL BY ROW` tables, as the previous recommendations were not valid. [#78676][#78676]
-- Added a `crdb_internal.validate_ttl_scheduled_jobs` builtin which verifies that each table points to a valid scheduled job which will action the deletion of expired rows. [#78373][#78373]
-- Adds a `crdb_internal.repair_ttl_table_scheduled_job` builtin, which repairs the given TTL table's scheduled job by supplanting it with a valid schedule. [#78373][#78373]
+ `ALTER SUPER REGION` follows the same rules as `ADD` or `DROP` super region. [#78462][#78462]
+
+- The [session variables](../v22.1/set-vars.html) `datestyle_enabled` and `intervalstyle_enabled`, and the [cluster settings](../v22.1/cluster-settings.html) `sql.defaults.datestyle.enabled` and `sql.defaults.intervalstyle.enabled` no longer have any effect. After upgrading to v22.1, these settings are effectively always interpreted as `true`. [#78455][#78455]
+- Bucket-count of [hash-sharded index](../v22.1/hash-sharded-indexes.html) is now shown from the `crdb_internal.table_indexes` table. [#78625][#78625]
```

Comment:
Bucket-count of [hash-sharded index](../v22.1/hash-sharded-indexes.html) --> 

`BUCKET_COUNT` for [hash-sharded indexes](../v22.1/hash-sharded-indexes.html)